nixnet/system/system.py

```diff
@@ -68,6 +70,11 @@ def close(self):
         self._handle = None
 
     @property
+    def databases(self):
```

Comment:
Possibly but we should dig deeper or call these thing out in the PR rather than deleting.